src/storage/src/hummock/state_store.rs

```diff
@@ -268,6 +268,9 @@ impl StateStore for HummockStorage {
             warn!("sealing invalid epoch");
             return;
         }
+        // Update `seal_epoch` synchronously,
+        // as `HummockEvent::SealEpoch` is handled asynchronously.
+        self.seal_epoch.store(epoch, MemOrdering::SeqCst);
```

Comment:
Before this PR, the seal epoch should be updated in somewhere else. Should we remove updating the seal epoch there?

Reply:
> Before this PR, the seal epoch should be updated in somewhere else. Should we remove updating the seal epoch there?

+1. I think we don't need to update seal epoch when handling `HummockEvent::SealEpoch` in `HummockEventHandler`.

Reply:
Fixed.